app/src/main/res/values/strings.xml

```diff
@@ -289,4 +289,19 @@
   <string name="wikidata_edit_failure">Failed to update corresponding Wikidata entity!</string>
   <string name="menu_set_wallpaper">Set wallpaper</string>
   <string name="wallpaper_set_successfully">Wallpaper set successfully!</string>
+  <string name="quiz">Quiz</string>
+  <string name="quiz_question_string">Is it Ok to Upload ? </string>
+  <string name="question">Question</string>
+  <string name="result">Result</string>
+  <string name="quiz_back_button">Do you want to end Quiz ? </string>
+  <string name="quiz_alert_message">Percentage of images reverted is more than 50%.Take Quiz to get familiar with the topicness and copyrights assosciated with creative commoms</string>
```

Comment:
I think we should probably make 50% a variable instead of hardcoding it into the string, as it could change. Aside from that, I would change this to: "More than 50% of the images you uploaded have been deleted. If you carry on uploading images that require deletion, your account will likely be banned. Would you like to take a quiz to help you learn what type of images you should or shouldn't upload?"